xml/System.Formats.Cbor/CborReader.xml

```diff
@@ -35,11 +35,15 @@
         <Parameter Name="allowMultipleRootLevelValues" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="data">To be added.</param>
-        <param name="conformanceMode">To be added.</param>
-        <param name="allowMultipleRootLevelValues">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="data">The CBOR encoded data to read.</param>
+        <param name="conformanceMode">Specifies a conformance mode guiding the checks performed on the encoded data.
+              Defaults to <see cref="F:System.Formats.Cbor.CborConformanceMode.Strict" /> conformance mode.</param>
```

Comment:
nit: consider using boilerplate text of enum param.
https://github.com/dotnet/dotnet-api-docs/wiki/Parameters%3A-Methods-and-constructors

Reply:
```suggestion
        <param name="conformanceMode">One of the enumeration values to specify a conformance mode guiding the checks performed on the encoded data.
              Defaults to <see cref="F:System.Formats.Cbor.CborConformanceMode.Strict" /> conformance mode.</param>
```